src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -69,6 +74,10 @@ class AuthScreens extends React.Component {
 
         Timing.start(CONST.TIMING.HOMEPAGE_INITIAL_RENDER);
         Timing.start(CONST.TIMING.HOMEPAGE_REPORTS_LOADED);
+
+        if (props.initialReportID) {
+            this.setInitialHomeParams(props.initialReportID);
+        }
```

Comment:
I think we could maybe simplify this a bit by just setting `this.initialReportID` instead of a method to do this for us?